lib/src/tile_map_parser.dart

```diff
@@ -65,8 +92,11 @@ class TileMapParser {
   static Iterable<XmlElement> _getPropertyNodes(XmlElement node) {
     var propertyNode = node.children
         .where((node) => node is XmlElement)
```

Comment:
I think you can call `cast<XmlElement>()` after the where on the stream and then you don't need the multiple casts following. (not sure though, would have to test)